### DISCUSSION TITLE
How I can update userAuth( store additional data ) ?

### DISCUSSION THREAD

**Discussion by kisscodes1909** [gladly-team/next-firebase-auth#253] [Q&A]
My users need more information like firstName, lastname, billing address, ...

My plan is to create a new document in firestore and update the information to 'userAuth'

I don't see any method to update it

Best,
Huu

**Comment by shreyas-jadhav** [comment]
Not sure if you want to update the database document or AuthUser object. But if you want to update the AuthUser, you can set custom claims or update the properties by using Admin SDK. I really recommend going through Firebase docs first.

[https://firebase.google.com/docs/auth/admin/manage-users](https://firebase.google.com/docs/auth/admin/manage-users)

**Comment by kisscodes1909** [comment]
Hi @shreyas-jadhav, I think claims are not good for store additional data

'Custom claims are only used to provide access control. They are not designed to store additional data (such as profile and other custom data). '

My plan create a user document in firestore and store additional data, But how do we add additional data to userAuth.

**Reply by vpaflah** [reply]
Did you find a way to store extra data from firestore doc, I'm also stuck at this point.

**Reply by lazycipher** [reply]
@vpaflah did you find any way for this?

**Comment by shreyas-jadhav** [comment]
I'm not saying using claims. You can update the already given properties. Like 
`AuthUser.displayName`, `AuthUser.phoneNumber` updating these wouldn't hurt anything as they are meant for it
